bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,230 @@ class StylintBear:
+        :param indent_size:
+            This works in conjunction with ``max_selector_depth``. If you
+            indent with spaces this is the number of spaces you indent with.
+            If you use hard tabs, set this value to ``0``.
+            Example if ``2``: prefer
+            ``/s/smargin: 0`` over ``/s/s/smargin: 0``
+        :param leading_zero:
+            When ``True``, prefer leading zeroes on decimal points. When
+            ``False``, disallow leading zeroes on decimal points.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param max_errors:
+            Set 'max' number of Errors. Use ``0`` to disable this feature.
+        :param max_warnings:
+            Set 'max' number of Warnings. Use ``0`` to disable this feature.
+        :param mixed_spaces_and_tabs:
+            Returns ``true`` if mixed spaces and tabs are found. If a
+            non-negative number is passed to ``indent_size``, it assumes soft
+            tabs (ie, spaces), and if ``0`` is passed to ``indent_size``
+            it assumes hard tabs.
+        :param variable_naming_convention:
+            Enforce a particular naming convention when declaring classes, ids,
+            and variables. Throws a warning if you don't follow the convention.
+            Supported values are ``lowercase-dash``, ``lowercase_underscore``,
+            ``camelCase``, ``BEM`` and ``False``.
+        :param class_naming_convention:
+            By default, ``variable_naming_convention`` only looks at variable
+            names. If ``class_naming_convention`` is set to ``true``,
+            ``variable_naming_convention`` will  also look at class and
+            id names.
+        :param none_keyword:
+            If ``True`` check for places where ``none`` used instead of 0. If
+            ``False`` check for places where 0 could be used instead of none.
+        :param check_no_important_keyword:
+            If ``true``, show warning when ``!important`` is found.
+        :param spaces_inside_parentheses:
+            Enforce or disallow use of extra spaces inside parentheses.
+        :param placeholder:
+            Enforce extending placeholder vars when using ``@extend(s)``.
+        :param prefix_vars_with_dollar:
+            Enforce use of ``$`` when defining a variable. In Stylus using a
+            ``$`` when defining a variable is optional, but is a good idea
+            if you want to prevent ambiguity. Not including the ``$`` sets up
+            situations where you wonder: "Is this a variable or a value?"
+            For instance: padding ``$default`` is easier to understand than
+            padding ``default``.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param semicolons:
+            When ``True``, allow semicolons. When ``False``, disallow
+            semicolons. When ``None``, do not check either way.
+            Supported values are ``True``, ``False`` and ``None``.
```

Comment:
> None is not supported from outside, please don't mention it. Say that by default either way is allowed without mentioning None.
We can just use None in the signature as default values bypass the type-check by coala.

I've used the default values on https://github.com/SimenB/stylint (best possible settings) and making things simpler for the user because all he has to do is just run coala without being worried. If we deviate from the defaults and use `None`, he might have to configure the `.coafile` manually and go through the docs. Your opinion ?